app/helpers/application_helper.rb

```diff
@@ -1,2 +1,47 @@
 module ApplicationHelper
+
+  def recreation_list
+    [
+      [' '],
+      ['Ski'],
+      ['Board'],
+      ['Both'],
+      ['Other']
+    ]
+  end
+
+  def trip_list
+    [
+      [' '],
+      ['One Way'],
+      ['Round Trip']
+    ]
+  end
+
+  def pass_list
+    [
+      [' '],
+      ['IKON'],
+      ['Epic'],
+      ['Other'],
+      ['No Preference'],
+      ['No Pass']
+    ]
+  end
+
+  def dates_list
+    [
+      [' '],
+      ['Monday'],
+      ['Tuesday'],
+      ['Wednesday'],
+      ['Thursday'],
+      ['Friday'],
+      ['Saturday'],
+      ['Sunday'],
+      ['Weekend Fri-Sun'],
+      ['Any Day']
```

Comment:
Instead of doing a 'Weekend Fri-Sun' option, would it be possible to make it a checkbox instead of a dropdown and then allow people to select multiple days if they want? Or is that going to be a lot of heavy lifting to allow for multiple days like that? 

Either that, or I think we should add a 'Weekend Sat-Sun' option for if you can only spend one night.

Reply:
Check box I think will just make for some different structures queries. An additional drop down would be easier for that purpose as this form final functionality will be to build active record queries. But I don't know enough about check boxes and passing in information in that  way to say is it is a difficult change. It could be just as easy.

Reply:
That works for me! I just know that sometimes people work late on Friday nights or it's hard to coordinate a Friday night departure, so if we're having a weekend option, I think just a two-day option would be nice too.